nekoyume/Assets/_Scripts/UI/Tween/AnchoredPositionSingleTweener.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using DG.Tweening;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Nekoyume.UI.Tween
+{
+    // todo: `AnchoredPositionSingleTweener`로 바꾸고, 좌표계를 선택할 수 있도록. `RotateSingleTweener` 참고.
+    [RequireComponent(typeof(RectTransform))]
+    public class AnchoredPositionSingleTweener : DOTweenBase
+    {
+        public enum AxisType
+        {
+            X,
+            Y,
+            Z
+        }
+
+        [SerializeField]
+        private float endValue = 0f;
+
+        [SerializeField]
+        private bool snapping = false;
+
+        [SerializeField]
+        private bool isFrom = false;
+
+        public Single single = Single.Z;
```

Comment:
If you not intended, default value is batter than specific value for field serialization.
```suggestion
        public Single single = Single.X;
```

Reply:
I put it in without much meaning, but I decided to delete the Z value, so I'll change it as suggested.